tw-tasks-kafka-listener-spring-boot-starter/src/main/java/com/transferwise/tasks/ext/kafkalistener/autoconfigure/TwTasksExtKafkaListenerAutoConfiguration.java

```diff
@@ -51,7 +50,6 @@ public KafkaListenerMetricsTemplate twTasksKafkaListenerMetricsTemplate(IMeterCa
   }
 
   @Bean
-  @Validated
```

Comment:
This properties is no longer validated? I can't see the explicit validator classes in this subproject.

Reply:
The whole module has been deprecated for a long time and should not be used. However codesearch shows it is still widely used in Wise.
I'll add the programmatic validation here as well.